src/Sentry.Unity.Editor/ConfigurationWindow/CoreTab.cs

```diff
@@ -23,10 +24,35 @@ internal static void Display(ScriptableSentryUnityOptions options)
                 new GUIContent("Capture In Editor", "Capture errors while running in the Editor."),
                 options.CaptureInEditor);
 
-            options.EnableLogDebouncing = EditorGUILayout.Toggle(
-                new GUIContent("Enable Log Debouncing", "The SDK debounces log messages of the same type if " +
-                                                        "they are more frequent than once per second."),
-                options.EnableLogDebouncing);
+            EditorGUILayout.Space();
+            EditorGUI.DrawRect(EditorGUILayout.GetControlRect(false, 1), Color.gray);
+            EditorGUILayout.Space();
+
+            {
```

Comment:
l: What is the deal with this block here?

Reply:
It's just visually blocking all those option blocks to make it easier skimming through.